package.json

```diff
@@ -26,6 +27,12 @@
     "when": "^1.7.1"
   },
   "devDependencies": {
+    "babel-cli": "^6.5.1",
```

Comment:
FWIW, `babel-cli` is useless here. It's meant to be installed globally (I personally just added `./node_modules/.bin` to my `$PATH`)